slf4j-api/src/main/java/org/slf4j/helpers/BasicMDCAdapter.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2004-2011 QOS.ch
+ * Copyright (c) 2004-2011 QOS.ch, Copyright (C) 2015 Google Inc.
```

Comment:
Since I work for Google, all our patches go by an opensource team and they requested that I add that.